packages/indexer-common/src/query-fees/models.ts

```diff
@@ -227,6 +279,69 @@ export function defineQueryFeeModels(sequelize: Sequelize): QueryFeeModels {
     { sequelize, tableName: 'vouchers' },
   )
 
+  ReceiptAggregateVoucher.init(
+    {
+      allocationId: {
+        type: DataTypes.CHAR(40), // 40 because prefix '0x' gets removed by TAP agent
+        allowNull: false,
+        primaryKey: true,
+        get() {
+          const rawValue = this.getDataValue('allocationId')
+          return toAddress(rawValue)
+        },
+        set(value: Address) {
+          const addressWithoutPrefix = value.toLowerCase().replace('0x', '')
+          this.setDataValue('allocationId', addressWithoutPrefix)
+        },
+      },
+      senderAddress: {
+        type: DataTypes.CHAR(40), // 40 because prefix '0x' gets removed by TAP agent
+        allowNull: false,
+        primaryKey: true,
+        get() {
+          const rawValue = this.getDataValue('senderAddress')
+          return toAddress(rawValue)
+        },
+        set(value: string) {
+          const addressWithoutPrefix = value.toLowerCase().replace('0x', '')
+          this.setDataValue('senderAddress', addressWithoutPrefix)
+        },
```

Comment:
Maybe you need to do the conversions to bigint using a similar code to this but for timestampNs and aggregateValue